src/App.js

```diff
@@ -92,7 +94,7 @@ function App() {
             <Routes>
               <Route path='/' element={<HomePage />} />
```

Comment:
You can also do like this:
`let routes = [
    {path: "/",element: HomePage},
    {path: "/signup",element: SignupPage},
    {path: "/doces",element: DocesPage},
    ...
]`
Then you can map and place it here

Reply:
Oh yeah! I will do that!